src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -518,9 +526,9 @@ private void validateCvrFileSources() {
             Logger.log(Level.SEVERE, "CDF files must be tabulated individually.");
           }
           if (!source.getFilePath().endsWith(".xml") && !source.getFilePath().endsWith(".json")) {
-            Logger.log(Level.SEVERE,
-                "Unexpected file extension: %s.  CDF source files must be .xml or .json",
-                source.getFilePath());
+            Logger
```

Comment:
You can delete this now (since you moved it to the "basic" check, though I recommend using the message you changed it to here for that one.